docs/content/en/docs/concepts/architecture/components/KLTcomponents.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Lifecycle Toolkit Components
+linktitle: KLT Components
+description: Basic understanding of Keptn Lifecycle Toolkit Components
+weight: 80
+cascade:
+---
+
+### Keptn Lifecycle Toolkit Components
+
+The Keptn Lifecycle Toolkit consists of two main components:
+
+* Keptn Lifecycle Operator, splits into two separate operators:
+  * Lifecycle-Operator
+  * Metrics-Operator
+* Keptn Lifecycle Scheduler
+
+```mermaid
+graph TD;
+    KLTComponents-->Operators;
+    KLTComponents-->Scheduler
+   Operators-->Lifecycle-Operator
+   Operators-->Metrics-Operator
+
+style KLTComponents fill:#006bb8,stroke:#fff,stroke-width:px,color:#fff
+style Operators fill:#d8e6f4,stroke:#fff,stroke-width:px,color:#006bb8
+style Scheduler fill:#d8e6f4,stroke:#fff,stroke-width:px,color:#006bb8
+style Lifecycle-Operator fill:#d8e6f4,stroke:#fff,stroke-width:px,color:#006bb8
+style Metrics-Operator fill:#d8e6f4,stroke:#fff,stroke-width:px,color:#006bb8
+
+```
+
+**Keptn's Lifecycle Operator** is
+a Kubernetes [operator](https://kubernetes.io/docs/concepts/extend-kubernetes/operator/)
+that automates the deployment and management
+of Keptn's Lifecycle Controller components in a Kubernetes cluster.
+The Keptn Lifecycle Operator contains several controllers for **Keptn CRDs**
+and a **Mutating Webhook**.
+
+Here's a brief overview:
+
+**Keptn CRDs:** The Keptn Lifecycle Operator contains
+several controllers that are responsible for managing
+and reconciling different types of Keptn CRDs, some of them are
+Project Controller,Service Controller,Stage Controller,and more.
+
+**Mutating Webhook:** It is used to automatically inject Keptn-specific labels
+and annotations into Kubernetes resources,
+such as deployments and services.
+These labels and annotations are used to enable Keptn's automation
+and monitoring capabilities.
+
+**Keptn's Lifecycle Metrics Operator** is
+a component of the Keptn platform that is responsible for collecting, processing,
+and analyzing metrics data from a variety of sources.
```

Comment:
```suggestion
**Keptn's Lifecycle Metrics Operator** 
collects,  processes, and analyzes metrics data from a variety of sources.
```
In a comma-separated list, you need a space after each comma before the next word.

I also don't think you need to say that this is a component of Keptn, just say what it does.